backend/pkg/logger/data/logger.go

```diff
@@ -143,7 +154,13 @@ func (sublogger *Logger) Stop() error {
 	}
 
 	for _, file := range sublogger.valueFileSlice {
-		file.Close()
+		err := file.Close()
+		if err != nil {
+			return logger.ErrClosingFile{
+				Name:      Name,
+				Timestamp: time.Now(),
+			}
+		}
```

Comment:
try to close all the files before returning any errors because this might leave multiple files open.
Also this usually returns error when something really bad is going on or where the file has already been closed, so I would just print the errors and call it a day